BigImageViewer/src/main/java/com/github/piasy/biv/view/BigImageView.java

```diff
@@ -312,6 +312,11 @@ public void showImage(Uri uri) {
     public void showImage(final Uri thumbnail, final Uri uri) {
         mThumbnail = thumbnail;
         mUri = uri;
+
+        if (mProgressIndicatorView != null && mProgressIndicatorView.getVisibility() == View.VISIBLE) {
+            removeView(mProgressIndicatorView);
```

Comment:
After we remove it, we should set `mProgressIndicatorView` to `null`.

Besides, I think we could change all `mProgressIndicatorView.setVisibility(GONE)` calls into removeView and nullify.

Reply:
Done!